dev-libs/libconfig/libconfig-9999.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF="1"
+inherit eutils autotools-multilib
+
+DESCRIPTION="Libconfig is a simple library for manipulating structured configuration files"
```

Comment:
This is copied from the portage main tree... Anyway, should I fix, commit and do a new pull request?